### PR TITLE
Update docs for `LPUSHX` and `RPUSHX`

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Follow up from https://github.com/aws/glide-for-redis/pull/1239#discussion_r1557959459

### CODE REVIEW

**Review by jonathanl-bq — APPROVED** [review]

File: java/client/src/main/java/glide/api/commands/ListBaseCommands.java [hunk 1]
```diff
@@ -325,8 +325,8 @@ CompletableFuture<Long> linsert(
     CompletableFuture<String[]> brpop(String[] keys, double timeout);
 
     /**
-     * Inserts specified values at the tail of the <code>list</code>, only if <code>key</code> already
-     * exists and holds a list.
+     * Inserts all the specified values at the tail of the list stored at <code>key</code>, only if
+     * the list already exists.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/commands/ListBaseCommands.java:329]: Split into two sentences. 
```
     * If <code>key</code> is a list, inserts all the specified values at the tail of the list. If
     * <code>key</code> is not a list, this performs no operation. 
```

Comment (acarbonetto) [reply java/client/src/main/java/glide/api/commands/ListBaseCommands.java:329]: I feel like we should explicitly specify that this performs no operation if the key is not a list.

File: java/client/src/main/java/glide/api/commands/ListBaseCommands.java [hunk 2]
```diff
@@ -341,8 +341,8 @@ CompletableFuture<Long> linsert(
     CompletableFuture<Long> rpushx(String key, String[] elements);
 
     /**
-     * Inserts specified values at the head of the <code>list</code>, only if <code>key</code> already
-     * exists and holds a list.
+     * Inserts all the specified values at the head of the list stored at <code>key</code>, only if
+     * the list already exists.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/commands/ListBaseCommands.java:345]: Split into two sentences.
```
     * If <code>key</code> is a list, inserts all the specified values at the head of the list. If
     * <code>key</code> is not a list, this performs no operation. 
```

**Review by acarbonetto — APPROVED** [review]

File: java/client/src/main/java/glide/api/commands/ListBaseCommands.java (hunk 1 above)
Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/ListBaseCommands.java]: updated

**Pull request merged**